docs/scalars.md

```diff
@@ -37,10 +37,44 @@ public class DateTimeScalar implements Coercing<LocalDateTime, String> {
 ```graphql
 scalar DateTime
 ```
+
 ## Registering Custom Scalars
-In more recent versions of `graphql-java` (>v15.0), some scalars, most notably, the `Long` scalar are no longer available by default.
-These are non standard scalar that is difficult for clients (e.g. JavaScript) to handle reliably.
-As a result of the deprecation,  you will need to 1) Define the scalar in your schema, and 2) Register the scalar.
+
+In more recent versions of `graphql-java` (>v15.0) some scalars, most notably the `Long` scalar, are no longer available by default.
+These are non standard scalar that are difficult for clients (e.g. JavaScript) to handle reliably.
+As a result of the deprecation, you will need to add them explicitly, and to do this you have a few options.
+
```

Comment:
Does it make sense to add the list of scalar extension types this would be applicable for? Or link to docs that list that out.